doc/advanced-configuration/outgoing-connections.md

```diff
@@ -213,20 +213,21 @@ A connection pool to ElasticSearch can be configured as follows:
 
 ```erlang
 {outgoing_pools, [
- {elastic, global, elasticsearch, [], [{host, "localhost"}]}
+ {elastic, global, default, [], [{host, "localhost"}]}
 ]}.
 ```
 
 MongooseIM uses [inaka/tirerl](https://github.com/inaka/tirerl) library to communicate with ElasticSearch.
 This library starts a pool of workers on its own so the following options are not configurable via `WPoolOpts`:
```

Comment:
Is it still valid, given the changes in `mongoose_wpool_elastic`? AFAIU now we're creating the pool on our own.

Reply:
It's true, however "starts a pool of workers on its own" is not exactly the direct reason anymore.

Reply:
True that :)